packages/core-contracts/contracts/proxy/OwnableUUPSImplementation.sol

```diff
@@ -0,0 +1,68 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./UniversalProxyImplementation.sol";
+import "../ownership/Ownable.sol";
+
+contract OwnableUUPSImplementation is UniversalProxyImplementation, Ownable {
```

Comment:
Rename `UniversalProxyImplementation` to `UUPSImplementation`?

Reply:
The UniversalProxyImplementation is storage agnostic. This Ownable (that doesn't need to be owneable according to the other thread) is the one that adds the unstructured storage